docker/deployed/testnet/api/config.json

```diff
@@ -44,7 +44,7 @@
     },
     "TelemetryPublisher": {
         "Enabled": true,
-        "MetricsHubURL": "https://metricshub-testnet-mrgr43cf5q-uw.a.run.app",
+        "MetricsHubURL": "https://metricshub-testnet-v2-mrgr43cf5q-uw.a.run.app",
```

Comment:
This new metric hub collects testnet metrics in the new BQ dataset.
We can think after the old is unused, to re-use the old non-v2 metrics hub again; maybe isn't worth it just for naming purity; just mentioning it.